submodules/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ endfunction()
 # Add taraxa-vdf
 set(VDF_LIBRARY_COMMAND "\
  ${CMAKE_MAKE_PROGRAM}\
- CPPFLAGS=\"-I./include -I${CONAN_INCLUDE_DIRS_OPENSSL} -I${CONAN_INCLUDE_DIRS_GMP} -I${CONAN_INCLUDE_DIRS_MPFR} -fPIC -std=c++20 -O3\"\
+ CPPFLAGS=\"-I./include -I${CONAN_INCLUDE_DIRS_OPENSSL} -I${CONAN_INCLUDE_DIRS_GMP} -I${CONAN_INCLUDE_DIRS_MPFR} -fPIC -std=c++2a -O3\"\
```

Comment:
Not 100% sure about this change but I cant build using -std=c++20, only c++2a works for me

Reply:
This is strange. What was wrong, do you have logs?

Reply:
```
/home/fornadel/.local/bin/cmake --build /home/fornadel/workspace/taraxa/taraxa-node/cmake-build-debug --target all -- -j 6
[  0%] Checking source code formatting by clang-format
[  2%] Analysing files by cppcheck ..
[  2%] Creating directories for 'vrf_lib_submodule'
[  2%] Creating directories for 'vdf_lib_submodule'
[  2%] Creating directories for 'secp256k1'
[  2%] Building CXX object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/ethash.cpp.o
[  2%] Building CXX object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/managed.cpp.o
[  3%] No download step for 'vrf_lib_submodule'
[  3%] No download step for 'vdf_lib_submodule'
[  3%] Performing download step (download, verify and extract) for 'secp256k1'
[  4%] Building C object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/keccak.c.o
-- Downloading...
   dst='/home/fornadel/workspace/taraxa/taraxa-node/cmake-build-debug/deps/src/secp256k1-ac8ccf29.tar.gz'
   timeout='none'
   inactivity timeout='none'
-- Using src='https://github.com/chfast/secp256k1/archive/ac8ccf29b8c6b2b793bc734661ce43d1f952977a.tar.gz'
[  4%] No update step for 'vdf_lib_submodule'
[  5%] No update step for 'vrf_lib_submodule'
[  5%] Building C object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/keccakf800.c.o
[  6%] No patch step for 'vdf_lib_submodule'
[  6%] No patch step for 'vrf_lib_submodule'
[  7%] Building C object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/keccakf1600.c.o
[  8%] Performing configure step for 'vdf_lib_submodule'
[  8%] Performing configure step for 'vrf_lib_submodule'
[  8%] Building C object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/primes.c.o
[  9%] Building CXX object submodules/CMakeFiles/ethash.dir/ethash/lib/ethash/progpow.cpp.o
[  9%] Performing build step for 'vdf_lib_submodule'
make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
[  9%] Linking CXX static library ../lib/libethash.a
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
g++: error: unrecognized command line option ‘-std=c++20’; did you mean ‘-std=c++2a’?
make[3]: *** [Makefile:35: obj/Hash2Prime.o] Error 1
[  9%] Built target ethash
make[2]: *** [submodules/CMakeFiles/vdf_lib_submodule.dir/build.make:86: deps/src/vdf_lib_submodule-stamp/vdf_lib_submodule-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:703: submodules/CMakeFiles/vdf_lib_submodule.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```